test/models/driver_test.rb

```diff
@@ -50,32 +51,68 @@
     it "must have a VIN number" do
       # Arrange
       new_driver.vin = nil
+      new_driver.save
 
       # Assert
       expect(new_driver.valid?).must_equal false
       expect(new_driver.errors.messages).must_include :vin
       expect(new_driver.errors.messages[:vin]).must_equal ["can't be blank"]
     end
+
+    it "must have a unique VIN number" do
+      # Arrange
+      new_driver.save
+      conflicting_driver = Driver.new(name: "Cathy", vin: "123", available: true)
+
+      # Assert
+      expect(conflicting_driver.valid?).must_equal false
+      expect(conflicting_driver.errors.messages).must_include :vin
+      expect(conflicting_driver.errors.messages[:vin]).must_equal ["has already been taken"]
+    end
+
   end
 
   # Tests for methods you create should go here
   describe "custom methods" do
     describe "average rating" do
-      # Your code here
+      it "calculates the average rating" do
```

Comment:
It might be worth using more white space (more empty lines), more helper methods, and maybe even many different `it` blocks to help organize these tests.